uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/ui/carddetailscheckout/CardDetailsCheckoutScreen.kt

```diff
@@ -81,7 +82,18 @@ internal fun CardDetailsCheckoutScreen(
     showDojoBrand: Boolean,
 ) {
     val state = viewModel.state.observeAsState().value ?: return
-    val keyboardController = LocalSoftwareKeyboardController.current
+    val view = LocalView.current
+    val keyboardController = object : KeyboardController {
+        val imm = LocalContext.current.getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
+
+        override fun show() {
+            imm?.showSoftInput(view, 0)
+        }
+
+        override fun hide() {
+            imm?.hideSoftInputFromWindow(view.windowToken, 0)
+        }
+    }
```

Comment:
I think we can use remember to abstract this part

Reply:
done 🚀